tests/NLog.UnitTests/Targets/FileTargetTests.cs

```diff
@@ -2280,14 +2282,21 @@ public void Dont_throw_Exception_when_archiving_is_enabled()
 </nlog>
 ");
 
-            NLog.LogManager.GetLogger("Test").Info("very important message");
+                NLog.LogManager.GetLogger("Test").Info("very important message");
+            }
+            finally
+            {
+                LogManager.Configuration = null;
```

Comment:
When I debug I see that [https://github.com/NLog/NLog/blob/master/tests/NLog.UnitTests/NLogTestBase.cs#L64](https://github.com/NLog/NLog/blob/master/tests/NLog.UnitTests/NLogTestBase.cs#L64) does work. In fact, I don't think this test was a problem. I changed this merely for good measure (good practice).

I think the actual problem is `AsyncTargetWrapperTests` (see next comment).